microservices-connector/cmd/main.go

```diff
@@ -79,8 +80,34 @@ func main() {
 		tlsOpts = append(tlsOpts, disableHTTP2)
 	}
 
+	webhookServiceName := controller.GetEnvWithDefault("SERVICE_NAME", "gmc-validating-webhook-service")
```

Comment:
Hi @ichbinblau, we may need to define a constant set for these environment variable, such as `SERVICE_NAME` and `NAMESPACE`

Reply:
Hi @zhlsunshine . Good catch. It has been resolved. PTAL.